### PR TITLE
Add interview guide example

### PR DESCRIPTION
**Pull request by smbuthia**
As we develop a supervisor reference app, we are using the resources and tools in that process to update and flesh out our design documentation. This interview guide example is a simplified version of the one that was used to design the supervisor reference app and will be linked as a resource in the reference app documentation.

### CODE REVIEW

File: content/en/design/guides/designing-interviews.md [hunk 1]
```diff
@@ -73,6 +75,60 @@ At the end of the interview, you wrap up in a way that makes participants feel a
 
 Things you should do include:
```
Comment (leah-ngaari) [inline content/en/design/guides/designing-interviews.md:76]: Might we missing the sub title "End" here?

File: content/en/design/guides/designing-interviews.md [hunk 2]
```diff
@@ -73,6 +75,60 @@ At the end of the interview, you wrap up in a way that makes participants feel a
...
+5. _What challenges have you experienced while compiling and reporting the data from CHWs?_
+6. _What improvements do you think would help improve the reporting process?_
+
+#### Ending the interview
+
+_We have come to the end of the interview. Do you have anything to add that you feel we have not covered?_
+
+_Thank you so much for taking the time to speak with me today. Learning about how you conduct supervision and reporting activities will really help us understand how to create solutions to support your work. If you think of anything else or you have any questions, you are more than welcome to get in touch. I also want to ask if we can contact you again if we think of other questions or if something is unclear. Is that alright?_
+
+{{% alert title="Observation checklist" %}}
```
Comment (leah-ngaari) [inline content/en/design/guides/designing-interviews.md:121]: We can remove this section on observation as it is a separate user research method

File: content/en/design/guides/designing-interviews.md (hunk 1 above)
Comment (smbuthia) [reply content/en/design/guides/designing-interviews.md:76]: It's still there.

File: content/en/design/guides/designing-interviews.md (hunk 2 above)
Comment (smbuthia) [reply content/en/design/guides/designing-interviews.md]: I've removed the observation checklist.

File: content/en/design/guides/designing-interviews.md (hunk 1 above)
Comment (leah-ngaari) [reply content/en/design/guides/designing-interviews.md:76]: Alright, I have seen it

**Review by leah-ngaari — APPROVED** [review]

**Pull request merged**